### PR TITLE
Factions: FVFHelp as HTMLBox

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by HoeenCoder** [comment]
Both ways are wrong, this should be programmed to work with the `/help` command.

**Comment by DeathlyPlays** [comment]
so make it work with /help fvf?
I was just fixing it up but I can do that

**Comment by HoeenCoder** [comment]
yes

**Comment by DeathlyPlays** [comment]
Done @HoeenCoder

File: wavelength-plugins/factions.js [hunk 1]
```diff
@@ -743,39 +743,34 @@ exports.commands = {
 			output += "</table></center>";
 			this.sendReplyBox(output);
 		},
-		'': 'help',
```
Comment (HoeenCoder) [inline wavelength-plugins/factions.js]: you can keep these, just make them parse the help command now, its useful for finding commands on what to do.

File: wavelength-plugins/factions.js [hunk 2]
```diff
@@ -995,25 +990,16 @@ exports.commands = {
 			delete Rooms.global.FvF[factionId];
 			delete targetRoom.fvf;
 		},
-
-		'': 'help',
```
Comment (HoeenCoder) [inline wavelength-plugins/factions.js]: ditto

File: wavelength-plugins/factions.js (hunk 1 above)
Comment (DeathlyPlays) [reply wavelength-plugins/factions.js]: Wouldn't it be simplier just having it /factionhelp (works as /help faction)

Comment (DeathlyPlays) [reply wavelength-plugins/factions.js]: Done

**Pull request merged**